src/hevm/src/EVM.hs

```diff
@@ -110,6 +114,7 @@ data VM = VM
   , _burned         :: Word
   , _constraints    :: [(SBool, Whiff)]
   , _iterations     :: Map CodeLocation Int
+  , _options        :: VMOpts
```

Comment:
I ended up adding the options struct to `VM`, since I needed to check at runtime whether or not `allowFFI` was set. Not sure if there's a cleaner way?

Reply:
hmm, well, we don't need the entire thing. Why don't we just have `ffiEnabled :: Bool` instead?

Reply:
yes true. we can always generalise to a `runtimeOpts` or similar if we need to later.